frontend/package.json

```diff
@@ -17,6 +17,7 @@
     "axios": "^1.6.7",
     "axios-hooks": "^5.0.2",
     "billboard.js": "^3.10.3",
+    "d3": "^7.8.5",
```

Comment:
do we actually use this? keeping because it's peer dep anyway and maybe we will decide to